modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/CheckEmptyCORSConfigurationsTestCase.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *
+ *   Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change the license header to WSO2 LLC.

Reply:
Changed https://github.com/wso2/product-apim/pull/12982/commits/89c0c1cb6ac7821d68a10e25d866de5b63fa815c